apps/angular/src/app/features/anime/pages/anime/anime-page.component.html

```diff
@@ -0,0 +1,45 @@
+@if (isLoading$ | async; as isLoading) {
+	<mat-progress-spinner color="primary" mode="indeterminate" class="spinner" diameter="36"></mat-progress-spinner>
+}
+
+@if (animes$ | async; as animes) {
+	<div class="table-wrapper">
+		<table mat-table [dataSource]="animes.items" class="mat-elevation-z8">
+
+			<ng-container matColumnDef="image">
+				<th mat-header-cell *matHeaderCellDef></th>
+				<td mat-cell *matCellDef="let anime">
+					<img src={{anime.image}} alt="{{anime.name}}" class="poster" />
+				</td>
+			</ng-container>
+
+			<ng-container matColumnDef="titleEnglish">
+				<th mat-header-cell *matHeaderCellDef> Title English </th>
+				<td mat-cell *matCellDef="let anime"> {{anime.titleEnglish}} </td>
+			</ng-container>
+
+			<ng-container matColumnDef="titleJapanese">
+				<th mat-header-cell *matHeaderCellDef> Title Japanese </th>
+				<td mat-cell *matCellDef="let anime"> {{anime.titleJapanese}} </td>
+			</ng-container>
+
+			<ng-container matColumnDef="airedStart">
+				<th mat-header-cell *matHeaderCellDef> Title Japanese </th>
+				<td mat-cell *matCellDef="let anime"> {{ anime.aired.start | date }} </td>
+			</ng-container>
+
+			<ng-container matColumnDef="type">
+				<th mat-header-cell *matHeaderCellDef> Type </th>
+				<td mat-cell *matCellDef="let anime"> {{anime.type}} </td>
+			</ng-container>
+
+			<ng-container matColumnDef="status">
+				<th mat-header-cell *matHeaderCellDef> Status </th>
+				<td mat-cell *matCellDef="let anime"> {{anime.status}} </td>
+			</ng-container>
+
+			<tr mat-header-row *matHeaderRowDef="displayedColumns; sticky: true"></tr>
```

Comment:
We place the description of the columns and the list of rows at the beginning of the table.